code/__DEFINES/dcs/signals.dm

```diff
@@ -653,6 +653,9 @@
 ///from base of mob/living/simple_animal/hostile/regalrat: (mob/living/simple_animal/hostile/regalrat/king)
 #define COMSIG_RAT_INTERACT "rat_interaction"
 
+///from /obj/item/slapper/attack_obj(obj/O, mob/living/user, params)
```

Comment:
No `obj/O`--https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names